config.js

```diff
@@ -0,0 +1,13 @@
+require('dotenv').config();
+
+function parseBoolean(boolToParse) {
+  if (boolToParse === 'true') return true;
+  if (boolToParse === 'false') return false;
+  return undefined;
+}
+
+module.exports = {
+  baseUrl: process.env.BASE_URL,
+  withAcl: parseBoolean(process.env.ACL),
+  language: process.env.DEFAULT_LANGUAGE,
+};
```

Comment:
`dotenv` allows for the environment variables which are declared in the .env pulled into this file at runtime.